### DISCUSSION TITLE
It would be great if we could..

### DISCUSSION THREAD

**Discussion by scripaca-alpaca** [victornpb/undiscord#202] [3. Ideas]
Delete specific user messages from every channel, not limited to one channel.

**Comment by ghost** [comment]
Undiscord already has the option to [delete specific messages](https://github.com/victornpb/deleteDiscordMessages/blob/master/help/filters.md) from a user.
 
![undiscord](https://user-images.githubusercontent.com/86576431/128029898-23053afe-bf68-450f-a983-eb0582ace013.png)

You can delete specific user messages [by specifying their ID](https://github.com/victornpb/deleteDiscordMessages/blob/master/help/authorId.md), the guild ID, and the ID for the specific channel in that guild.

**Reply by victornpb** [reply]
I suggest clicking the (?) in front of the fields, they will forward you to the instructions page about each option
